src/main/java/org/dcsa/ovs/service/impl/TimestampServiceImpl.java

```diff
@@ -56,6 +72,8 @@
         String modeOfTransport = timestamp.getModeOfTransport() != null ? timestamp.getModeOfTransport().name() : null;
 
         return transportCallRepository.getTransportCall(timestamp.getUNLocationCode(), timestamp.getFacilitySMDGCode(), modeOfTransport, timestamp.getVesselIMONumber())
+                // We should create the transport call if it is missing, but we are missing that feature.
+                .switchIfEmpty(Mono.error(new ResponseStatusException(HttpStatus.NOT_IMPLEMENTED)))
```

Comment:
Wouldn't creating TransportCall be part of this PR?

Reply:
No, it requires the new publisher layout as we need carrier SMDG code.